packages/tailwindcss/src/css-functions.test.ts

```diff
@@ -136,6 +136,27 @@ describe('theme function', () => {
         `)
       })
 
+      test('theme(colors.red.500/75%)', async () => {
```

Comment:
Same test as below, but without spaces. Already worked, but added for completeness sake.